pkg/hive/install.go

```diff
@@ -167,6 +169,11 @@ func (c *clusterManager) clusterDeploymentForInstall(doc *api.OpenShiftClusterDo
 		}
 	}
 
+	clusterDomain := doc.OpenShiftCluster.Properties.ClusterProfile.Domain
+	if !strings.ContainsRune(clusterDomain, '.') {
+		clusterDomain += "." + os.Getenv("DOMAIN_NAME")
```

Comment:
```suggestion
		if envDomainName, ok := os.LookupEnv("DOMAIN_NAME"); ok {
			clusterDomain += "." + envDomainName
		} else {
			// return some sort of error if the env var is unset
			// this ensures we don't return this with an empty string
		}
```

I think I'd still feel more comfortable if we were using `m.env.Domain()` so that we had a standardized call and that one already has error checking too, but I'm okay with this.

Reply:
The Hive managed doesn't have access to `Domain()` so we'll need to refactor it a bit. Since we want to get this deployed to test before the deploy freeze, I'd say that we should do that in a follow up PR as well.